tests/door43_tools_tests/test_templaters.py

```diff
@@ -48,9 +48,9 @@ def testTemplaterTaComplete(self):
         test_file_path = self.extractZipFiles(test_folder_name)
         success = self.doTemplater('ta', test_file_path)
         self.verifyTaTemplater(success, expect_success, self.out_dir,
-                               ['checking.html', 'intro.html', 'process.html', 'translate.html'])
+                               ['1-intro.html', '2-process.html', '3-translate.html', '4-checking.html'])
```

Comment:
This test appears to not be testing anything. I reverted the changes to `client/preprocessors.py` and this test still passes. The test in `test_ta_preprocessor.py` did fail, as expected.